src/index.html

```diff
@@ -8,6 +8,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars  stars--0">
```

Comment:
you don't need to add more than one space between words, fix all cases
```suggestion
    <div class="stars stars--0">
```